packages/calcite-components/src/utils/floating-ui.ts

```diff
@@ -406,6 +413,18 @@ export function getEffectivePlacement(floatingEl: HTMLElement, placement: Logica
   return placement.replace(/leading/gi, placements[0]).replace(/trailing/gi, placements[1]) as EffectivePlacement;
 }
 
+const hiddenFloatingElStyle = {
+  visibility: "hidden",
+  pointerEvents: "none",
+  // initial positioning based on https://floating-ui.com/docs/computePosition#initial-layout
```

Comment:
This seems to be working fine. It doesn't seem to need `top` or `left`. We do have `inset` defined so maybe that's why. We really just want the element to be off the screen and not affecting any layout when its not open.